content/docs/sidebar.json

```diff
@@ -5,6 +5,29 @@
     "source": "index.md",
     "icon": "house"
   },
+  {
+    "label": "CML",
+    "slug": "cml",
+    "source": "cml/index.md",
+    "icon": "cml",
+    "children": [
+      "usage",
+      "get-started",
+      {
+        "label": "CML with DVC",
+        "slug": "cml-with-dvc"
+      },
+      {
+        "label": "Self-hosted runners",
+        "slug": "self-hosted-runners"
+      },
+      {
+        "label": "Install CML as a package",
```

Comment:
```suggestion
        "label": "Install CML as a Package",
```

Reply:
But the slug and doc title are "CML with NPM". It's best to be consistent for SEO I think.